scripts/server.mjs

```diff
@@ -25,7 +26,12 @@ app.prepare().then(async () => {
           req.url === "/" ||
           req.url.startsWith("/ready");
 
-        return !isNext;
+        const isLegacy = () => {
```

Comment:
```
const isLegacy = req.url.search(/^\/visualizations\/[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}\b\/image/)

return !isNext || isLegacy
```

I'd be more precise for the regex and I think we don't have to pass by a temporary variable. WDYT ?

Reply:
I agree with it